File: src/Tawtheef.Websites/operations-web/src/app/pages/user/employee/exams-management/models/exam-lookups.dto.ts
export interface ExamLookupsDto {
  categories: ExamLookupItemDto[];
  interruptionPolicies: ExamLookupItemDto[];
  specializedCategoryId: string;
}

export interface ExamLookupItemDto {
  id: string;
  nameAr: string;
  nameEn: string;
}

File: src/Application.Operation/Features/Employee/Exams/DTOs/ExamLookupsDto.cs
namespace Application.Operation.Features.Employee.Exams.DTOs;

public sealed record ExamLookupsDto(List<ExamLookupItemDto> Categories,
    List<ExamLookupItemDto> InterruptionPolicies, Guid SpecializedCategoryId);

public sealed record ExamLookupItemDto(Guid Id, string NameAr, string NameEn);
